### ISSUE TITLE
Block interaction help for Clay Oven says "Light fire" when there is food inside

### ISSUE THREAD

**Issue by rowantwig** [anegostudios/VintageStory-Issues#3735]
### Game Version

v1.19.4

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

After you've put food in an oven, the info says "Light fire", suggesting you set the food on fire as if it was firewood. This is bad advice and may trick new players into burning their food for no reason. The info should give you a different message (or no message at all) when there is food inside rather than firewood.

### How to reproduce

1. Build a Clay Oven.
2. Follow the on-screen instructions to the letter.
3. Burn your food as a result.

### Screenshots

![2024-03-19_11-53-31](https://github.com/anegostudios/VintageStory-Issues/assets/162151342/8bc48818-8f58-4963-aa58-39801621feee)

### Logs

```Log```

**Comment by Craluminum2413** [comment]
@rowantwig Does interaction change if you look away and then look at block again?

**Comment by rowantwig** [comment]
@Craluminum2413 No, it still says "Light fire" if I look away and look back at the oven.

**Comment by pingouyn** [comment]
Sounds like a duplicate of #3731?  New users set food on fire in the clay oven because the tooltips are not explicit enough.
